Composer/packages/client/src/pages/language-understanding/index.tsx

```diff
@@ -154,33 +120,19 @@ const LUPage: React.FC<DefineConversationProps> = props => {
       </div>
       <div css={ContentStyle} data-testid="LUEditor">
         <div css={projectContainer}>
-          <Tree variant="large" css={projectTree}>
-            <div css={projectWrapper}>
-              <Nav
-                onLinkClick={(ev, item) => {
-                  item && onSelect(item.id);
-                  ev && ev.preventDefault();
-                }}
-                styles={{
-                  root: {
-                    /* override dulplicate selected mark bellow All*/
-                    selectors: {
-                      'ul>li>ul button.ms-Nav-chevronButton:after': {
-                        borderLeft: 'none',
-                      },
-                    },
-                  },
-                  chevronButton: {
-                    backgroundColor: 'transparent',
-                  },
+          {navLinks.map(dialog => {
```

Comment:
can we reuse the leftNav component defined in lg page ? they looks like  have the same constructure.